src/http.py

```diff
@@ -0,0 +1,78 @@
+###Finding records in Sierra that do not have working links###
+
+import csv
+import requests
+from pymarc import Field,Subfield,MARCReader
+
+class Rowish:
+    def __init__(self,control,url,bib_code,suppressed,live):
+        self.control = control,
+        self.url = url
+        self.bib_code = bib_code,
+        self.suppressed = suppressed,
+        self.live = live
+
+def csv_classer(file):
+    classed = []
+    for row in file:
+        new_row = Rowish(row[0],row[1],row[2],False,True)
+        classed.append(new_row)
+    return classed
+
+def mrc_classer(file):
+    classed = []
+    for record in file:
+        control = record.get_fields["001"]
```

Comment:
that syntax does not look correct - will need to test it.
`pymarc` allows accessing the first encountered MARC tag using dictionary syntax: record["001"] - this returns `pymarc.field` instance.
`record.get_fields` is a method so probably the correct syntax should be `record.get_fields("091")` and it will return a list of fields.